### ISSUE TITLE
Lottie animation don't show on first load

### ISSUE THREAD

**Issue by georgeemr** [MPowerKit/Popups#1]
I'm using the SkiaSharp.Extended.UI.Controls to show a Lottie animation. On the first load, the Lottie doesn't show. But if you close and reopen, the animation shows successfully.

https://www.nuget.org/packages/SkiaSharp.Extended.UI.Maui

Note: The maui community toolkit popup works on the first call.

```
<?xml version="1.0" encoding="utf-8" ?>
<popup:PopupPage x:Class="DemoApp.Views.PopUps.LoadingMopup"
                 xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
                 xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
                 xmlns:popup="clr-namespace:MPowerKit.Popups;assembly=MPowerKit.Popups"
                 xmlns:skia="clr-namespace:SkiaSharp.Extended.UI.Controls;assembly=SkiaSharp.Extended.UI"
                 Title="PopupTestPage"
                 BackgroundInputTransparent="False"
                 BackgroundColor="#80000000"
                 CloseOnBackgroundClick="True">
    <Frame BackgroundColor="WhiteSmoke"
           VerticalOptions="CenterAndExpand"
           WidthRequest="200">
        <VerticalStackLayout HorizontalOptions="Center"
                             VerticalOptions="Center">           
            <skia:SKLottieView RepeatCount="-1"
                               RepeatMode="Reverse"
                               Source="AnimationBunny.json"
                               WidthRequest="150"
                               HeightRequest="150" />
            <Label HorizontalOptions="Center"
                   x:Name="lblMessage"
                   VerticalOptions="Center"
                   HorizontalTextAlignment="Center"></Label>
        </VerticalStackLayout>
    </Frame>
</popup:PopupPage>
```

**Comment by Alex-Dobrynin** [comment]
Sorry for so late response. Lost notification about this issue.

May I ask you to provide repro project and indicate the OS in which you experiencing this issue

**Comment by georgeemr** [comment]
Sure, this is the repo:

https://1drv.ms/u/s!AlTIxW_ExT1enfJaJq2TRvReR-oLrw?e=Zkb9hq

I'm running on Android 14

**Comment by Alex-Dobrynin** [comment]
![image](https://github.com/MPowerKit/Popups/assets/23138430/e6f8f8f0-de5f-4d98-9135-b242ad1dfd80)
You missed up the mauiprogram file

**Comment by Alex-Dobrynin** [comment]
I've found that you opened similar issue on SkiaSharp extended github
https://github.com/mono/SkiaSharp.Extended/issues/264
and there you are using community toolkit popup, and you wrote there that it does not work. so i have question, do animation work in community toolkit popup or no?

Seems, that this is SkiaSharp.Extended.Maui bug. because in your example there is label in the popup and if you comment the lottieanimation view, then you will see the label and other content, but if you uncomment lottie animation view, then you will not see even the label

**Comment by georgeemr** [comment]
Hi Alex, that issue is different because the Lottie doesn't render as expected, but it opens on the first show on the community toolkit.
The issue with Mopup is that the Lottie doesn't show at all on the first show.
I also realized that if you change a text label on the first show, neither change either.

**Comment by Alex-Dobrynin** [comment]
This is not Mopups library. Btw, did you triedwith mopups library, is there the same behavior?

**Comment by georgeemr** [comment]
Sorry for the confusion Alex! You are right; indeed I'm using your library, not mopups. After all the research my decision was to use this lib.
I will try with mopups and let you know.

**Comment by Alex-Dobrynin** [comment]
fixed in 1.1.5 nuget version

**Comment by Alex-Dobrynin** [comment]
@georgeemr by the way, recently created new library for lottie animations using native libraries
[MPowerKit.Lottie](https://github.com/MPowerKit/Lottie), you may try to use it, it has more options than skiasharp
